src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -56,6 +61,12 @@ const DetailsSidebar = ({
 
     return (
         <SidebarContent hasTitle={hasTitle} title={<FormattedMessage {...messages.sidebarDetailsTitle} />}>
+            {versions &&
```

Comment:
Should be inside its own file, say `SidebarVersions` and here it will be
`{hasVersions && <SidebarVersions versions={versions} onVersionHistoryClick={onVersionHistoryClick}>}`